### ISSUE TITLE
Import XML fails with 413 status code for "POST /sites/200317893/imports/new

### ISSUE THREAD

**Issue by kosiew** [Automattic/wp-calypso#58559]
### Quick summary

Customer is exported a file from a GoDaddy WordPress site.
The XML file is included in #4552001-zen

The import fails with this error:

![image](https://user-images.githubusercontent.com/29057562/143811408-1d2369af-baae-47ec-ad92-11494ce6d368.png)

### Steps to reproduce

1. Start at Tools > Import on a Simple Site
2. Import the xml file

### What you expected to happen

Import to complete successfully

### What actually happened

Fails

### Context

#4552001-zen

### Reproducibility

Consistent

### Severity

One

**Comment by kosiew** [comment]
@vishnugopal , 
Can you take a look?

**Comment by kosiew** [comment]
Resolved here - #4567728-zen

TLDR; 
The source GoDaddy site was no longer active or accessible to user -> import failed.

**Comment by vishnugopal** [comment]
Interesting that the solution here is All in one Importer -> temporarily upgraded business plan for site -> import -> downgrade site to Premium.
